src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -547,7 +546,7 @@ private void InitNativeProcess()
 
             // Send Telemetry indicating what argument passing mode we are in.
             ApplicationInsightsTelemetry.SendExperimentalUseData(
-                ExperimentalFeature.PSWindowsNativeCommandArgPassing,
+                "PSWindowsNativeCommandArgPassing",
```

Comment:
hmm, this appears again, so it seems intentional. Why do we want to continue to send this telemetry?

Reply:
these features are still interesting to us to help us understand how the system is being used. We don't have another API to send usage telemetry, so we're continuing to use (and need) this.